src/metabase/search/config.clj

```diff
@@ -186,57 +186,59 @@
   "All of the result components that by default are displayed by the frontend."
   #{:name :display_name :collection_name :description})
 
-(defmulti searchable-columns-for-model
-  "The columns that will be searched for the query."
-  {:arglists '([model])}
-  (fn [model] model))
+(defmulti searchable-columns
+  "The columns that can be searched for each model."
+  {:arglists '([model search-native-query])}
```

Comment:
This function now allows us to descriminate between cases when `{:search-native-query true}` is in the search context or not.

The purpose is to allow us to only score non-native query columns [here](https://github.com/metabase/metabase/pull/43982/files#diff-7f2653f8c0f476e8c282b17d0512711d156e14a3cdc2204465b841490bc2ff8bR161) when `{:search-native-query true}` is not in the search context.